src/accelerate/utils/environment.py

```diff
@@ -13,7 +13,21 @@
 # limitations under the License.
 
 import os
-from distutils.util import strtobool
+
+
+def str_to_bool(value) -> bool:
```

Comment:
Just curious why the same implementation is not used as for `strtobool`? The latter is a bit faster, but it shouldn't matter in practice.